libvast/vast/error.hpp

```diff
@@ -85,6 +85,8 @@ enum class ec : uint8_t {
   silent,
   /// Insufficient memory.
   out_of_memory,
+  /// An error occured
```

Comment:
This description needs to be a bit more meaningful.

Reply:
Huh, not sure what happened to the rest of the sentence